### PR TITLE
fix: UI issues

### PR DESCRIPTION
**Pull request by vivek-devtron**
# Description

This PR contains the multiple ui fixes

Fixes [AB#2892](https://dev.azure.com/DevtronLabs/Devtron/_workitems/edit/2892)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Manually tested by opening all the cases in specific order

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by sohel-rp — APPROVED** [review]

File: src/components/hostURL/HostURL.tsx [hunk 1]
```diff
@@ -161,7 +161,7 @@ export default class HostURLConfiguration extends Component<HostURLConfigProps,
         return (
             <>
                 <section
-                    className="mt-16 mb-16 ml-20 mr-20 global-configuration__component"
+                    className="global-configuration__component"
```
Comment (shivani170) [inline src/components/hostURL/HostURL.tsx:164]: Classname `global-configuration__component` is not making any sense in this component

File: src/components/project/ProjectList.tsx [hunk 2]
```diff
@@ -187,7 +187,7 @@ export default class ProjectList extends Component<ProjectListProps, ProjectList
             return <ErrorScreenManager code={this.state.code} />
         } else {
             return (
-                <section className="mt-16 mb-16 ml-20 mr-20 global-configuration__component flex-1">
+                <section className="global-configuration__component flex-1">
```
Comment (shivani170) [inline src/components/project/ProjectList.tsx:190]: same here

File: src/components/cluster/Cluster.tsx [hunk 3]
```diff
@@ -212,7 +212,7 @@ export default class ClusterList extends Component<ClusterListProps, any> {
                 'Clusters' +
                 (this.props.serverMode === SERVER_MODE.EA_ONLY || window._env_.K8S_CLIENT ? '' : ' and Environments')
             return (
-                <section className="mt-16 mb-16 ml-20 mr-20 global-configuration__component flex-1">
+                <section className="global-configuration__component flex-1">
```
Comment (shivani170) [inline src/components/cluster/Cluster.tsx:215]: Classname global-configuration__component is not making any sense in this component. Please rename it to the respective component

File: src/components/dockerRegistry/Docker.tsx [hunk 4]
```diff
@@ -92,7 +92,7 @@ useEffect(() => {
     dockerRegistryList = [{ id: null }].concat(dockerRegistryList)
     return (
         <section
-            className="mt-16 mb-16 ml-20 mr-20 global-configuration__component flex-1"
+            className="global-configuration__component flex-1"
```
Comment (shivani170) [inline src/components/dockerRegistry/Docker.tsx:95]: same here

File: src/components/gitOps/GitOpsConfiguration.tsx [hunk 5]
```diff
@@ -458,7 +458,7 @@ class GitOpsConfiguration extends Component<GitOpsProps, GitOpsState> {
             )
         }
         return (
-            <section className="mt-16 mb-16 ml-20 mr-20 global-configuration__component flex-1">
+            <section className="global-configuration__component flex-1">
```
Comment (shivani170) [inline src/components/gitOps/GitOpsConfiguration.tsx:461]: Same here

File: src/components/gitProvider/GitProvider.tsx [hunk 6]
```diff
@@ -122,7 +122,7 @@ export default function GitProvider({ ...props }) {
 
     return (
         <section
-            className="mt-16 mb-16 ml-20 mr-20 global-configuration__component flex-1"
+            className="global-configuration__component flex-1"
```
Comment (shivani170) [inline src/components/gitProvider/GitProvider.tsx:125]: same here

**Review by shivani170 — APPROVED** [review]

File: src/components/hostURL/HostURL.tsx (hunk 1 above)
Comment (vivek-devtron) [reply src/components/hostURL/HostURL.tsx:164]: it is existing so didn't change this

**Pull request merged**